react-carousel-component/my-app/src/App.js

```diff
@@ -0,0 +1,21 @@
+// import logo from './logo.svg';
+import './App.css';
+import Carousel from './Carousel';
+
+const images = [
+  '/images/001.png',
+  '/images/004.png',
+  '/images/007.png',
+  '/images/025.png',
+  '/images/039.png',
+];
+
+function App() {
+  return (
+    <div className="App">
+      <Carousel itemsArray={images}></Carousel>
```

Comment:
It's best not to encode the data type in the prop name. It's redundant and actually makes it harder to modify the code. (If you decide to use an object now you have to rename all the places `itemsArray` is used.)